core/network/src/debug_info.rs

```diff
@@ -287,16 +287,15 @@ where TSubstream: AsyncRead + AsyncWrite {
 				Async::NotReady => break,
 				Async::Ready(NetworkBehaviourAction::GenerateEvent(event)) => {
 					match event {
-						IdentifyEvent::Identified { peer_id, info, .. } => {
+						IdentifyEvent::Received { peer_id, info, .. } => {
 							self.handle_identify_report(&peer_id, &info);
 							let event = DebugInfoEvent::Identified { peer_id, info };
 							return Async::Ready(NetworkBehaviourAction::GenerateEvent(event));
 						}
-						IdentifyEvent::Error { .. } => {}
-						IdentifyEvent::SendBack { result: Err(ref err), ref peer_id } =>
+						IdentifyEvent::Error { peer_id, error } =>
 							debug!(target: "sub-libp2p", "Error when sending back identify info \
```

Comment:
```suggestion
							debug!(target: "sub-libp2p", "Identification with peer {:?} failed => {}", peer_id, error)
```
`IdentifyEvent::Error` is no longer specific to errors occurring during sending of replies.